Fix line-endings-dependent tests for non-*NIX systems

```
......................................................

52 passing (8s)
2 failing

1) lines EachPos:
   AssertionError: 388 === 393
    at testEachPosHelper (D:\Docs\Projects\Web\recast\test\lines.js:73:12)
    at exports.testEachPos (D:\Docs\Projects\Web\recast\test\lines.js:89:5)
    at Context.<anonymous> (D:\Docs\Projects\Web\recast\test\run.js:14:21)
    at Test.Runnable.run (D:\Docs\Projects\Web\recast\node_modules\mocha\lib\runnable.js:194:15)
    at Runner.runTest (D:\Docs\Projects\Web\recast\node_modules\mocha\lib\runner.js:372:10)
    at D:\Docs\Projects\Web\recast\node_modules\mocha\lib\runner.js:448:12
    at next (D:\Docs\Projects\Web\recast\node_modules\mocha\lib\runner.js:297:14)
    at D:\Docs\Projects\Web\recast\node_modules\mocha\lib\runner.js:307:7
    at next (D:\Docs\Projects\Web\recast\node_modules\mocha\lib\runner.js:245:23)
    at Object._onImmediate (D:\Docs\Projects\Web\recast\node_modules\mocha\lib\runner.js:274:5)
    at processImmediate [as _immediateCallback] (timers.js:330:15)

2) lines CharAt:

    + expected - actual

    +<CR>
    -

    at Lines.compare (D:\Docs\Projects\Web\recast\test\lines.js:108:16)
    at Lines.Lp.eachPos (D:\Docs\Projects\Web\recast\lib\lines.js:497:17)
    at exports.testCharAt (D:\Docs\Projects\Web\recast\test\lines.js:113:11)
    at Context.<anonymous> (D:\Docs\Projects\Web\recast\test\run.js:14:21)
    at Test.Runnable.run (D:\Docs\Projects\Web\recast\node_modules\mocha\lib\runnable.js:194:15)
    at Runner.runTest (D:\Docs\Projects\Web\recast\node_modules\mocha\lib\runner.js:372:10)
    at D:\Docs\Projects\Web\recast\node_modules\mocha\lib\runner.js:448:12
    at next (D:\Docs\Projects\Web\recast\node_modules\mocha\lib\runner.js:297:14)
    at D:\Docs\Projects\Web\recast\node_modules\mocha\lib\runner.js:307:7
    at next (D:\Docs\Projects\Web\recast\node_modules\mocha\lib\runner.js:245:23)
    at Object._onImmediate (D:\Docs\Projects\Web\recast\node_modules\mocha\lib\runner.js:274:5)
    at processImmediate [as _immediateCallback] (timers.js:330:15)
```

--- a/test/lines.js
+++ b/test/lines.js
@@ -81,8 +81,10 @@ function testEachPosHelper(lines, code) {
 }
 
 exports.testEachPos = function(t) {
-    var code = arguments.callee + "",
-        lines = fromString(code);
+    // Function.prototype.toString uses \r\n line endings on non-*NIX
+    // systems, so normalize those to \n characters.
+    var code = (arguments.callee + "").replace(/\r\n/g, "\n");
+    var lines = fromString(code);
 
     testEachPosHelper(lines, code);
 
@@ -99,8 +101,10 @@ exports.testEachPos = function(t) {
 };
 
 exports.testCharAt = function(t) {
-    var code = arguments.callee + "",
-        lines = fromString(code);
+    // Function.prototype.toString uses \r\n line endings on non-*NIX
+    // systems, so normalize those to \n characters.
+    var code = (arguments.callee + "").replace(/\r\n/g, "\n");
+    var lines = fromString(code);
 
     function compare(pos) {
         assert.strictEqual(

